src/components/screens/reclaimBalance/modal/summary/index.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import to from 'await-to-js';
+import { useSelector } from 'react-redux';
+import { selectAccount } from '@store/selectors';
+import { create } from '@api/transaction';
+import { tokenMap, MODULE_ASSETS_NAME_ID_MAP } from '@constants';
+import { useTransactionFeeCalculation, useTransactionPriority } from '@shared/transactionPriority';
+import AccountMigration from '@shared/accountMigration';
+import TransactionSummary from '@shared/transactionSummary';
+import LiskAmount from '@shared/liskAmount';
+import { toRawLsk } from '@utils/lsk';
+import styles from './summary.css';
+
+const moduleAssetId = MODULE_ASSETS_NAME_ID_MAP.reclaimLSK;
+
+const Summary = ({
+  nextStep,
+  t,
+}) => {
+  const token = tokenMap.LSK.key;
+  const account = useSelector(selectAccount);
+  const network = useSelector(state => state.network);
+  const [
+    // eslint-disable-next-line no-unused-vars
+    selectedPriority, _, priorityOptions,
+  ] = useTransactionPriority(token);
+
+  const { minFee } = useTransactionFeeCalculation({
+    selectedPriority,
+    token,
+    account: account.info.LSK,
+    priorityOptions,
+    transaction: {
+      moduleAssetId,
+      nonce: account.info.LSK.sequence.nonce,
+      senderPublicKey: account.info.LSK.summary.publicKey,
+      amount: account.info.LSK.legacy.balance,
+    },
+  });
+
+  const onSubmit = async () => {
+    const data = {
+      moduleAssetId,
+      network,
+      senderPublicKey: account.info.LSK.summary.publicKey,
```

Comment:
We can clean this up by creating a variable for senderPublicKey above and then using it in `useTransactionFeeCalculation ` and then in `onSubmit`. Same for other values